wellen/inputs

```diff
@@ -0,0 +1 @@
+../inputs
```

Comment:
@1024bees Could you please remove the symlink and change the path in the CI settings instead?
Otherwise, this is good to go.

Reply:
done with https://github.com/ekiwi/wellen/pull/17/commits/50390d78828e04fecf8e4998d612498e66bf5fbd, thanks for all your patience :)